changelogs/current_change_log.md

```diff
@@ -5,8 +5,46 @@
 ---
 
 
+### Tables added
+
+#### PlannedMovement
+
+Tracks the final history for each planned movement within the hospital.
+
+| Name | Type | Description |
+|---| --- |---|
+| planned_movement_id | bigint | Unique identifier in EMAP for this PlannedMovement record. |
+| hospital_visit_id | HospitalVisit | Identifier for the HospitalVisit associated with this record. |
+| location_id | Location | Planned Location  to move to, may be null. |
+| event_type | varchar(255) | /brief The type of planned movement event (ADMIT, TRANSFER, DISCHARGE). |
```

Comment:
Do we want the `/brief` in these descriptions?

Reply:
oh interesting, we don't, but I thought they were picked up by the documentation only - is there something different in the new config file?

Reply:
noice